lib/gitsh/argument_list.rb

```diff
@@ -0,0 +1,21 @@
+module Gitsh
+  class ArgumentList
+    def initialize(args)
+      @args = args
+    end
+
+    def length
+      args.length
+    end
```

Comment:
Where is `#length` used? The only place I see it in this PR, aside from the unit tests, are in `ArgumentBuilder::combined_arguments` -- but that's for an Array created in the initializer, not for an ArgumentList.